courier/courier.go

```diff
@@ -147,10 +160,28 @@ func (m *Courier) DispatchQueue(ctx context.Context) error {
 			} else {
 				gm.SetAddressHeader("From", from, fromName)
 			}
+
 			gm.SetHeader("To", msg.Recipient)
 			gm.SetHeader("Subject", msg.Subject)
 			gm.SetBody("text/plain", msg.Body)
-			gm.AddAlternative("text/html", msg.Body)
+
+			tmpl, err := NewEmailTemplateFromMessage(m.d.Config(ctx), msg)
+			if err != nil {
+				m.d.Logger().
+					WithError(err).
+					WithField("message_id", msg.ID).
+					Error(`Unable to get email template from message.`)
```

Comment:
So we send an empty body on error?

Reply:
Right, so with this configuration we'd send the email with just the plaintext version if we're unable to load the HTML template for some reason.

Reply:
Ahhh, I see - that makes sense!